### PR TITLE
Fixed reference to Trac in R1/Beta3 announcement

### PR DESCRIPTION
**Pull request by panos**

### CODE REVIEW

File: content/news/2021-06-14_haiku_r1beta3_timeline_approved.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ tags = ["haiku", "software", "release", "beta3", "timeline", "schedule"]
 +++
 
 The release timeline for Haiku R1/Beta 3 has been approved after a 7 day RFC (Request for Comment) period on the mailing list.
-{{< alert-info "Release timeline" "[**Access the release timeline on the Trac wiki.**](https://dev.haiku-os.org/wiki/R1/Beta3/Timeline). Note that minor changes to the dates may occur.">}}
+{{< alert-info "Release timeline" "<a href="https://dev.haiku-os.org"> target="_blank"><b>Access the release timeline on the Trac wiki.</b></a>. Note that minor changes to the dates may occur.">}}
```
Comment (nielx) [inline content/news/2021-06-14_haiku_r1beta3_timeline_approved.md:10]: But now it does not link to the actual timeline.

Comment (nielx) [reply content/news/2021-06-14_haiku_r1beta3_timeline_approved.md:10]: Also the HTML is wrong (stray > in there)

Comment (panos) [reply content/news/2021-06-14_haiku_r1beta3_timeline_approved.md:10]: SORRY

**Comment by panos** [comment]
Nope, I did it wrong again.

**Comment by panos** [comment]
This should be it.

**Pull request merged**